.mailmap

```diff
@@ -1492,3 +1492,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+Ishan Pandhare <91841626+Ishanned@users.noreply.github.com>
```

Comment:
The .mailmap file needs to be in alphabetical order. If you run the `bin/mailmap_check.py` script it will reorder .mailmap for you.